### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by SeanCarrick**
Working on the repository to, hopefully, get some contributors. @jkovalsky could you help with this document by providing the Asset Compiling section and the sections from Testing on down. Also, feel free to make any other changes you see fit. No rush on this, by the way.

-SC

### CODE REVIEW

**Review by jkovalsky — APPROVED** [review]
Finished my part hence approving for merging to master.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+- Wiki: For flow and class diagrams for the Project, go to the [Wiki](https://github.com/PekinSOFT-Systems/LoadMaster/wiki) for the repository.
+
+## Assets compilation
+
+Information about compiling CSS, JS, SVG, etc.
+
+## Environment setup
+
+When setting up your environment to contribute to PekinSOFT™ Systems projects, we ask that you *fork* the repository for the project in which you are interested. We strive to make sure that any required libraries are included in the repository, typically at the root level. If you have any issues running the project after you have forked it, please let us know by submitting an [Issue](https://github.com/PekinSOFT-Systems/LoadMaster/issues/new?assignees=SeanCarrick+jkovalsky&labels=won%27t+run&template=forked_issue.md&title=%5BRunning+Issue%5D) and we will try to assist with the configuration.
```
[the remark this answers is not in the document]
Comment (jkovalsky) [reply CONTRIBUTING.md]: The link simplified.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,15 +16,18 @@ Information about compiling CSS, JS, SVG, etc.
...
 
 ## Testing
 
+Since we at PekinSOFT™ Systems consider quality as one of the most important features of the software we built, we protect our main codebase from regressions already during pull request reviews. Automatic checks consist of WhiteSource Bolt security tests of the source code and Selenium based tests of the application behavior. No pull requests heading to master branch with failed checks will be accepted unless the such pull request gets exceptional approval from the project team. Any contribution adding brand new UI components like menus, dialogs, settings etc. must be accompanied by corresponding functional test.
```
[the remark this answers is not in the document]
Comment (jkovalsky) [reply CONTRIBUTING.md]: "the" removed.

**Comment by SeanCarrick** [comment]
Merging now...

**Pull request merged**